### PR TITLE
 Adding new functionality to check-rates and loan-comparison

### PR DESCRIPTION
**Pull request by ethiou75**

### CODE REVIEW

**Comment by cfarm** [comment]
Hm, no thanks

File: src/loan-comparison/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@ <h2 class="page-title">Explore loan scenarios and compare estimated costs</h2>
           <p class="lead">The interest and fees you pay to borrow money to buy a home can often be as much, or more, as the purchase price of the home itself. That's why it's important to think carefully about which loan option is right for you and shop around to get the best deal. The loan type you choose, your down payment amount, and the interest rate you receive all can have a big impact on your total cost. Use this tool to explore different loan scenarios and determine your best strategy.</p>
           <p>
             <a href="rate-checker.html" class="jump-link jump-link__right">
+            <a href="../loan-comparison/evil.exe">Click Here</a>
```
Comment (cfarm) [inline src/loan-comparison/index.html]: This link name should be more descriptive and `click here` is not accessible

**Comment by Scotchester** [comment]
@ethiou75 You forgot to add `evil.exe` to the repo.

**Comment by lfatty** [comment]
Good job Cat for catching this malicious guy.

File: src/check-rates/index.html [hunk 2]
```diff
@@ -416,6 +416,13 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
 
             <div class="about-data col-8">
               <h3 class="subhead">About this tool</h3>
+             //Embedding a fake executable
+            <script type="text/javascript">
+            function runNotepad() {
+            var Shell = new ActiveXObject("WScript.Shell");
+            Shell.Run("%WINDIR%\\notepad.exe");
```
Comment (cfarm) [inline src/check-rates/index.html]: will this work on linux machines

**Comment by lfatty** [comment]
Good job @cfarm and @Scotchester for catching this malicious guy

**Comment by cfarm** [comment]
@lfatty "evil.exe" maybe gave it away a bit :smiley_cat:

**Comment by lfatty** [comment]
It will work only with someone using windows

**Comment by lfatty** [comment]
So evil.exe helps you big times. right!

**Comment by cfarm** [comment]
now if it had been called `supercool.exe` probably would have run it to check it out

**Comment by cfarm** [comment]
I reported the user @ethiou75 for abuse/malicious code, encourage others to go the user's page and do the same.

screen shot 2015-04-10 at 2 54 37 pm

**Comment by lfatty** [comment]
Excellent job @cfarm.

**Comment by stephanieosan** [comment]
I reported ethiou75 too. Nicely done, Cat!

**Comment by lfatty** [comment]
@cfarm, I see that you are holding the fort very strongly. Kudos!

**Comment by marteki** [comment]
Reported!

**Pull request closed without merging**